### DISCUSSION TITLE
Otter styles priority

### DISCUSSION THREAD

**Discussion by decodist** [Codeinwp/otter-blocks#1578] [Q&A]
Is it possible to move Otter styles to the head using a hook or theme option? 

`<link rel='stylesheet' id='otter-post-v2-75-1678710623.css-css' ...`

At the moment, this is loaded in `wp_footer` which can cause some serious FOUT.

**Comment by HardeepAsrani** [comment]
Hey @decodist,

It's not possible right now to use an easy hook but we're working on some CSS related changes which should take care of it. But for now, it's not possible.

**Reply by decodist** [reply]
Hi @HardeepAsrani - have there been any recent changes to the way CSS is handled? Thanks

**Reply by HardeepAsrani** [reply]
@decodist Not recently. Has there been any issues on your side?

**Reply by decodist** [reply]
@HardeepAsrani yes, there have been issues since August 2022. https://github.com/Codeinwp/otter-blocks/issues/1133
